tests/integration/targets/autoscaling_instance/tasks/tests.yml

```diff
@@ -0,0 +1,5 @@
+---
+- ansible.builtin.include_tasks: describe.yml
+- ansible.builtin.include_tasks: attach_detach.yml
+- ansible.builtin.include_tasks: health.yml
+- ansible.builtin.include_tasks: protection.yml
```

Comment:
With permission changes it's getting further, but these files don't exist so it's failing here, now.

Reply:
Uploaded now, sorry.